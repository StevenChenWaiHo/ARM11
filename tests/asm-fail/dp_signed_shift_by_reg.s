; Signed shift register not supported in data processing instructions
sub r1, r2, -r3, lsl r4
