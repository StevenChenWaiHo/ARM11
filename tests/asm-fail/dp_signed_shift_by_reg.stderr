tests/asm-fail/dp_signed_shift_by_reg.s:2:14: Expected TOKEN_IDENT, but got TOKEN_MINUS(-)
